.github/workflows/github-page.yml

```diff
@@ -45,6 +45,21 @@ jobs:
         sudo apt-get -y remove --purge software-properties-common
         sudo apt-get -y autoremove
 
+    - name: Install secp256k1 (Linux)
+      if: matrix.os == 'ubuntu-latest'
+      run: |
+        sudo apt-get -y install autoconf automake libtool
+        mkdir secp256k1-sources
+        cd secp256k1-sources
+        git clone https://github.com/bitcoin-core/secp256k1.git
+        cd secp256k1
+        git reset --hard $SECP256K1_REF
```

Comment:
You forgot to specify the env variable: `SECP256K1_REF`. I've also made this change in #3745 due to your request. So it's up to you if you want to fix this PR or close it in favor of the aforementioned one

Reply:
Ah yes, thanks for the pointer.

Closing in favour of https://github.com/input-output-hk/cardano-node/pull/3745.